addon/components/base/bs-modal/dialog.js

```diff
@@ -134,6 +136,44 @@ export default Component.extend({
     return isBlank(size) ? null : `modal-${size}`;
   }).readOnly(),
 
+
+   /**
+   * The id of the `.modal-title` element
+   *
+   * @property titleId
+   * @type string
+   * @default null
+   * @private
+   */
+  titleId: null,
+
+/**
+   * Gets or sets the id of the title element for aria accessibility tags
+   *
+   * @method getSetTitleID
+   * @private
+   */
+  getOrSetTitleId() {
+      //Title element may be set by user so we have to try and find it to set the id
+      const modalNode = document.getElementById(this.get('id'));
```

Comment:
Isn't this the same as `this.get('element')`? Would be simples than searching the DOM...